tests/test_examples.py

```diff
@@ -112,7 +112,7 @@ def test_download_example_data_true_download():
 
 def test_failed_download(running_test):
     filename = "non_existing_file"
-    with pytest.raises(RuntimeError):
+    with pytest.raises(ValueError):
```

Comment:
You would have to change this if you follow my previous comment.

Reply:
Solved in [2235a19](https://github.com/ansys/pymapdl/pull/3474/commits/2235a190a8a01b944981549a3cc2af6d3621aa60)